### ISSUE TITLE
Pre-Rebase features still missing

### ISSUE THREAD

**Issue by IamVelcroboy** [DeltaV-Station/Delta-v#1181]
## Description

- [X] Shipyard
- [ ] Onierophage 
- [ ] Golems + Soul Crystals + Sacrificing (#1883) + Golemancy
- [x] Glimmer Wisps - #1383
- [ ] Shogi
- [ ] Drones
- [x] Fugitives - #1413
- [ ] Decloners
- [ ] Digging(this should be attempted upstream first)
- [X] Pitbulls - not wanted, https://github.com/DeltaV-Station/Delta-v/pull/1835#issuecomment-2357982974

Anything I missed?

**Comment by DangerRevolution** [comment]
> ## Description
> * [ ]  Shipyard
> * [ ]  Onierophage
> * [ ]  Golems + Soul Crystals + Sacrificing + Golemancy
> * [ ]  Glimmer Whisps
> * [ ]  Shogi
> * [ ]  Drones
> 
> Anything I missed?

Stamina-based combat

**Comment by IamVelcroboy** [comment]
> Stamina-based combat

There should be an admin discussion on that one. I can add it to the list if it's confirmed as wanted.

**Comment by Squishy77** [comment]
Arachnee

**Comment by IamVelcroboy** [comment]
> Arachnee

Again, not sure if we actually plan to incorporate that? I think we went in favor of upstream spider people. I could be wrong though. Get me an admin/maintainer to say otherwise and I'll add to the list.

**Comment by deltanedas** [comment]
lamia i think

**Comment by DangerRevolution** [comment]
> lamia i think

I think the running theory is that Null will port Arachne & Lamia from EE

**Comment by Squishy77** [comment]
Oh, dispel doesn't work on portals anymore and invisibility is bugged.

**Comment by rosieposieeee** [comment]
dispel also doesnt work on mime walls right now

tbh i dont miss stam combat. it made melee, which is already disadvantaged enough as is, even worse for no real reason

**Comment by IamVelcroboy** [comment]
> Oh, dispel doesn't work on portals anymore and invisibility is bugged.

> dispel also doesnt work on mime walls right now

These might just be better as their own separate issues.

**Comment by rosieposieeee** [comment]
yeah thats fair. im not sure what we are doing vis a vis psionics, if vm is still working on the rework for EE and we are gonna port that or what, but i assume we need that for oneiro, golems, and glimmer wisps

**Comment by Yousifb26** [comment]
Glimmer wisps

**Comment by Bonktrauma** [comment]
Shipyard is baack

**Comment by Bonktrauma** [comment]
EE is about done fixing oniro and arachne aswell

**Comment by Bonktrauma** [comment]
https://github.com/Simple-Station/Einstein-Engines/pull/438

**Comment by deltanedas** [comment]
> > Stamina-based combat
> 
> There should be an admin discussion on that one. I can add it to the list if it's confirmed as wanted.

did this ever have a discussion @therealDLondon

**Comment by therealDLondon** [comment]
> > > Stamina-based combat
> > 
> > 
> > There should be an admin discussion on that one. I can add it to the list if it's confirmed as wanted.
> 
> did this ever have a discussion @therealDLondon

No not yet, I can add it to the admin discussion for Saturday

**Comment by deltanedas** [comment]
i just remembered pitbulls :trollface:

**Comment by Bonktrauma** [comment]
> i just remembered pitbulls :trollface:

Felinids everywhere beware

**Comment by deltanedas** [comment]
and poor couriers

:s PITBULLS COMING FROM THE VENTS FUCK

**Comment by Squishy77** [comment]
Decoloners and fugtives- De-cloners did 16 genetic damage as a tier three sci tech and lived in a little case which could be emaged to be opened/ only openable with hos, warden or cap. 

Fugtives you know well.

**Comment by IamVelcroboy** [comment]
> Fugtives you know well.

What exactly happened to fugis? An upstream update broke them again? We had them for a little bit.

**Comment by deltanedas** [comment]
i dont think we ever had them added post rebase - https://github.com/DeltaV-Station/Delta-v/pull/193 was closed

**Comment by IamVelcroboy** [comment]
> i dont think we ever had them added post rebase - #193 was closed

Maybe that's what it was. Or VM was working on something before they left maybe 🤷‍♂️

**Comment by Bonktrauma** [comment]
> i dont think we ever had them added post rebase - https://github.com/DeltaV-Station/Delta-v/pull/193 was closed

Think we should re-open that? Debug is the one who closed it

**Comment by IamVelcroboy** [comment]
> Think we should re-open that? Debug is the one who closed it

If someone is going to work on it, sure. It was closed as "derelict" though.

**Comment by deltanedas** [comment]
that pr is old as fuck and im working on it using the new antag toys rn anyway

**Comment by IamVelcroboy** [comment]
Dope

**Comment by Bonktrauma** [comment]
> that pr is old as fuck and im working on it using the new antag toys rn anyway

Awesome

**Comment by Vapetastic-Gaming** [comment]
Depreciated Issue
